setup.rst

```diff
@@ -223,48 +363,34 @@ with **Homebrew**::
 
     $ brew install openssl xz
 
-and configure python versions >= 3.7::
+and ``configure`` Python versions >= 3.7::
 
     ./configure --with-pydebug --with-openssl=$(brew --prefix openssl)
 
-or configure python versions < 3.7::
+or ``configure`` Python versions < 3.7::
 
     $ CPPFLAGS="-I$(brew --prefix openssl)/include" \
       LDFLAGS="-L$(brew --prefix openssl)/lib" \
       ./configure --with-pydebug
 
-and make::
+and ``make``::
 
     $ make -s -j2
 
 or **MacPorts**::
 
     $ sudo port install pkgconfig openssl xz
 
-and configure::
+and ``configure``::
 
     $ CPPFLAGS="-I/opt/local/include" \
       LDFLAGS="-L/opt/local/lib" \
       ./configure --with-pydebug
 
-and make::
+and ``make``::
 
     $ make -s -j2
 
-
-This will build CPython with only warnings and errors being printed to
-stderr and utilize up to 2 CPU cores. If you are using a multi-core machine
-with more than 2 cores (or a single-core machine), you can adjust the number
-passed into the ``-j`` flag to match the number of cores you have.
-
-Do take note of what modules were **not** built as stated at the end of your
-build. More than likely you are missing a dependency for the module(s) that
-were not built, and so you can install the dependencies and re-run both
-``configure`` and ``make`` (if available for your OS).
-Otherwise the build failed and thus should be fixed (at least with a bug being
-filed on the `issue tracker`_).
-
```

Comment:
I removed this part because it was duplicated from above.  People that reach this point should have already read and know what ``make`` does.